.releaserc.json

```diff
@@ -6,9 +6,14 @@
       "prerelease": "rc"
     },
     {
-      "name": "release-v11",
+      "name": "release-v12",
```

Comment:
`master` should also publish to the `latest` channel (without `prerelease`).

Reply:
Also, don't forget to adjust the `release-v11` branch.

Reply:
Made `master` to be `latest`(https://github.com/GetStream/stream-chat-react/pull/2506/commits/6cc4f7ca77e3026f99cdd5f2faeb0689dc0f06c1)

Reply:
Made `release-v11` to be a version branch (https://github.com/GetStream/stream-chat-react/pull/2507)